kolibri/plugins/learn/options.py

```diff
@@ -1,9 +1,9 @@
 option_spec = {
     "Learn": {
-        "KOLIBRI_ENABLE_CUSTOM_CHANNEL_NAV": {
+        "ENABLE_CUSTOM_CHANNEL_NAV": {
             "type": "boolean",
             "default": False,
-            "envvars": ("KOLIBRI_ENABLE_CUSTOM_CHANNEL_NAV",),
+            "description": "Whether to enable custom channel navigation applications",
```

Comment:
@rtibbles Does `applications` here refer to plug-ins that might be added to Kolibri?

Reply:
Not exactly no, they are HTML5 apps that are able to do some navigation of content in Kolibri.